pkg/actions/airflow.go

```diff
@@ -55,7 +55,7 @@ var (
 	errAirflowHookDAGFailed     = errors.New("airflow hook DAG failed")
 )
 
-func NewAirflowHook(h ActionHook, action *Action, cfg Config, endpoint *http.Server) (Hook, error) {
+func NewAirflowHook(h ActionHook, action *Action, cfg Config, endpoint *http.Server, stats.Collector) (Hook, error) {
```

Comment:
missing the arg name - collector or statsCollector

Reply:
fixed 👍